docs/datasync/conflict-resolution-intro.md

```diff
@@ -88,17 +88,18 @@ const {
   modelDefs,
   {
     db,
-    dataSyncConflictMap: {
-      Comment: {
-        enabled: true,
-        deltaTTL: 604800
+    conflictConfig: {
+      configMap:{
```

Comment:
Change to something more obvious..Models?

Reply:
Thanks!